_data-prepper/pipelines/configuration/sources/s3.md

```diff
@@ -1,21 +1,61 @@
 ---
 layout: default
-title: Amazon S3 source
+title: `s3` source
```

Comment:
Remove tic marks for metadata titles - this will not allow the page to display properly.